e2e/tests/a11y/desktop/a11y-snapshot-test-guest.spec.js

```diff
@@ -57,6 +52,9 @@ test.describe('Accessibility Tests with Snapshots for guest user', () => {
     test('Product Detail Page should not have new accessibility issues', async ({page}) => {
         await navigateToPDPDesktop({page})
 
+        await expect(page.getByRole('heading', {name: /Cotton Turtleneck Sweater/i})).toBeVisible()
+        await expect(page.getByText(/From \$39\.99/i).nth(1)).toBeVisible()
+
```

Comment:
This will ensure the page is finished loading product detail data before running a11y. This will reduce the flakiness we've been seeing on nightly run